docs/src/main/sphinx/release/release-471.md

```diff
@@ -0,0 +1,66 @@
+# Release 471 (19 Feb 2025)
+
+## General
+
+* Add [](/functions/ai) for textual tasks on data using OpenAI, Anthropic, or
+  other LLMs using Ollama as backend.  ({issue}`25028`)
+* Include split count and total split distribution time in the `EXPLAIN ANALYZE`
+  output. ({issue}`25028`)
+* Add support for JSON logging format to console with `log.console-format=JSON`. ({issue}`25081`)
+* Add further packages for use with Python user-defined functions. ({issue}`25058`)
+* Improve performance for Python user-defined functions. ({issue}`25058`)
+* Improve performance for queries using top N operations. ({issue}`24937`)
+* Prevent failures when using fault-tolerant execution with an exchange manager
+  using Azure Storage with workload identity. ({issue}`25063`)
+
+## Server RPM
+
+* Remove RPM package. Use the tarball or container image instead, or build an
+  RPM with the setup in the [trino-packages
+  repository](https://github.com/trinodb/trino-packages). ({issue}`24997`)
+
+## Security
+
+* Fix initialization of the Ranger access control system with additional
+  configuration files. ({issue}`24887`)
+
+## Delta Lake connector
+
+* Add support for reading `variant` type. ({issue}`22309`)
+* Add [](/object-storage/file-system-local). ({issue}`25006`)
+* Support reading of `p`-type deletion vectors in Delta Lake. ({issue}`24946`)
+* Increase compatibility with S3-compatible storage solutions. ({issue}`24954`)
+* Add support for configuring `s3.storage-class` when writing objects to S3. ({issue}`24698`)
+* Fix query failures when writing to Delta Lake tables with statistics. ({issue}`25011`)
```

Comment:
In my understanding, this is not a query failure because the connector suppresses checkpoint query failure.
We could remove this entry, or change to "Fix failures when writing large checkpoint files" or something.

Reply:
Ah .. thanks. I misread the issue. Reworded